### ISSUE TITLE
eslint --init remains stuck at "Are you using ECMAScript 6 features?" under Node 8.1.0

### ISSUE THREAD

**Issue by ronjouch** [eslint/eslint#8702]
**Environment**

* **ESLint:** 3.19
* **Node:** 8.1.0
* **npm:** 5.0.3
* **OS/Terminal:** Bug happens under both:
    * Ubuntu 16.04.2LTS / GNOME Terminal 3.18.3 / {zsh 5.3.1, bash 4.3.48}
    * Windows 7 / cmd

**Reproduction steps**

```bash
~ npm install eslint
~ ./node_modules/.bin/eslint --init

? How would you like to configure ESLint? Answer questions about your style
? Are you using ECMAScript 6 features? (y/N)
```

**What did you expect to happen?**

Be able to type y/n

**What actually happened?**

* Typing produces nothing visible on screen.
* It is impossible to quit the assistant; Ctrl+C / Ctrl+D don't respond. I have to kill the terminal.

**Comment by Xisabla** [comment]
I have exactly the same problem:

ESLint: 3.19.0
Node: 8.1.0
npm: 5.0.3
OS:  Ubuntu 17.04
Terminal: Default linux terminal/zsh 5.2

Here is a gif of the problem: [http://imgur.com/TCp7NLW](http://imgur.com/TCp7NLW) (I'm spamming the enter key)

**Comment by avdv** [comment]
Same here.
**ESLint**: 3.19.0
**Node**: v8.1.0
**yarn**: 0.24.6

Attaching strace to the process, looks pretty normal:

```
strace: Process 5413 attached
epoll_wait(5, [{EPOLLIN, {u32=12, u64=12}}], 1024, -1) = 1
read(12, "\r", 65536) # hit Enter
epoll_wait(5, [{EPOLLIN, {u32=12, u64=12}}], 1024, -1) = 1
read(12, "n", 65536) # hit "n"
...
```

**Comment by hongchangwu** [comment]
Same problem.

*ESLint*: 3.19.0
*Node*: 8.1.0
*npm*: 5.0.3
*OS*: Windows 10

**Comment by soda0289** [comment]
Possibly related to this issue:
https://github.com/SBoudrias/Inquirer.js/issues/547

**Comment by Xisabla** [comment]
Yes that's it, I tested with node v8.0.0 it works.

**Comment by not-an-aardvark** [comment]
This should be fixed in Node 8.1.1, which was released today.

**Comment by ronjouch** [comment]
> *This should be fixed in Node 8.1.1, which was released today.*

Fix confirmed, using Node 8.1.1 from nvm under Ubuntu 16.04.2. Thanks for the followup @not-an-aardvark !

**Comment by toby5box** [comment]
Just a me-too, I had this problem on OS X with node v8.1.0, fixed in 8.2.1 (Homebrew).
